common/src/chain/config/builder.rs

```diff
@@ -13,7 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use super::emission_schedule::{self, *};
+use super::emission_schedule::*;
```

Comment:
I'm a big opponent of this kind of imports... very confusing and makes it hard to read files.

Reply:
I agree, fixed.